app/Resources/views/themes/tastefulplanet/layouts/1.html.twig

```diff
@@ -1,5 +1,5 @@
-<div data-studio-zones-container>
-    <div class="landing-page__zone landing-page__zone--{{ zones[0].id }}" data-studio-zone="{{ zones[0].id }}">
+<div class="landing-page__zones">
```

Comment:
This might be confusing for developers. `landing-page` is a content type. Maybe something more generic like `page` would work better.
What do you think guys?

Reply:
This is just a CSS class name, that is not used anywhere in the Page Builder app. Feel free to rename it however you want :)